snap/snapcraft.yaml

```diff
@@ -166,14 +179,35 @@ parts:
       - gstreamer1.0-libav
       - dbus-x11
       - projectm-data
+      - qtwayland5 # Attempt at Wayland Support
```

Comment:
Do these attempts work? If not, let's remove them. I very much doubt the Spotify one works but I don't know much about Wayland.

Reply:
Latest version released shows logs that it's looking for a "`libspotify`". As I don't have a paid account, I have no way of validating whether it works. That said, the previous errors are gone. See: https://github.com/kz6fittycent/clementine/issues/10

Reply:
The Wayland attempt defintely works (I'll remove the comment), @kz6fittycent will have to speak on the Spotify attempt.

Reply:
@kz6fittycent The libspotify.so is there, but there are broken symlinks. I have documented my findings in [kz6fittycent/clementine#10](https://github.com/kz6fittycent/clementine/issues/10)